src/GrassPokemon.java

```diff
@@ -0,0 +1,41 @@
+package src;
+
+public class GrassPokemon extends Pokémon {
+    private String razorLeaf;
+    private boolean grassyTerrain;
+
+    public GrassPokemon(String sound, String food, String razorLeaf, boolean grassyTerrain) {
+        super(sound, food);
+        this.razorLeaf = razorLeaf;
+        this.grassyTerrain = grassyTerrain;
+    }
+
+    public String getRazorLeaf() {
+        return razorLeaf;
+    }
+
+    public boolean isGrassyTerrain() {
```

Comment:
Heel goed dat je voor boolean een "is..." methode maakt ipv en "get..." methode.